docs/setup/call-a-service.md

```diff
@@ -57,7 +57,7 @@ The `snet` CLI tool is your swiss army knife for working with SingularityNET. It
 You can install the CLI with pip:
 
 ```
-pip install snet-cli
+pip3 install snet-cli
```

Comment:
Rustam's issue is also about a missing `sudo` in front of this command. Since we haven't asked people to create a virtualenv first, we probably should use sudo here.

Reply:
Done.